### DISCUSSION TITLE
Connecting to Google Cloud Mysql using SSL

### DISCUSSION THREAD

**Discussion by ralvs** [prisma/prisma#3075] [Q&A] [answered]
Hello guys!
I'm trying to connect my prisma2 project on my machine to Gcloud

Using this url it works normally: `mysql://user:password@ip:3306/prisma2`

But, when I add the SSL part it doesn't work: `mysql://user:password@ip:3306/prisma2?sslmode=require&sslcert=server-ca.pem&sslidentity=client-identity.p12&sslpassword=sslPassword`

By the way, from the command line I can also connect correctly with: `mysql -uroot -p -h <ip> --ssl-ca=server-ca.pem --ssl-cert=client-cert.pem --ssl-key=client-key.pem`

I have generated the .p12 file and the ssPassword as described at https://www.prisma.io/docs/reference/database-connectors/mysql 

Any ideias? :)
Thanks

**Comment by hi019** [comment, marked as answer]
Maybe https://github.com/prisma/prisma/issues/1228?

**Comment by ralvs** [comment]
Ok @hi019, so it's a known bug.
I will follow that issue
Thanks

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
